src/types/navigation.ts

```diff
@@ -144,8 +148,21 @@ export type AccountScreensStackParams = {
     onSuccess: (value?: void | PromiseLike<void> | undefined) => void;
     onCancel: (reasons?: any) => void;
   };
+  SettingsScreens: NavigatorScreenParams<SettingsScreensStackParams>;
 };
 
+export type SettingsScreensStackParams = {
+  Settings: undefined;
+  DisplayMode: undefined;
+  JoinCommunity: undefined;
+  CheckOldPassword: undefined;
+  CreateNewPassword: {
+    wallet?: LocalWallet;
```

Comment:
Wath do you think to add a `mode` field that tells if we are requesting the old password or creating a new?
This maybe can help you to better decide wath to do in the screen logic since can happe that the user don't have a `LocalWallet` when the account has been inported with a Ledger.

Reply:
I didn't understand what you want to achieve here. 
We already have two different screens based on what you wrote:
`CheckOldPassword`
`CreateNewPassword`
They render the same `ChangeWalletPassword` component.

Do you want me to rename the `CheckOldPassword` or create a third screen rendered by the same component?

Reply:
Oh, sorry didn't notice you already did how I was suggesting.